colors/termschool.vim

```diff
@@ -124,3 +124,6 @@ hi cssBraces ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
 hi SpellBad ctermfg=0 ctermbg=224 guifg=#000000 guibg=#ffdfdf
 hi SpellCap ctermfg=0 ctermbg=81 guifg=#000000 guibg=#5fd7ff
 hi SpellRare ctermfg=0 ctermbg=225 guifg=#000000 guibg=#ffdfff
+hi TabLineFill ctermfg=NONE ctermbg=NONE cterm=NONE guifg=NONE guibg=NONE gui=NONE
+hi TabLineSel ctermfg=NONE ctermbg=59 cterm=NONE guifg=NONE guibg=#3f4b52 gui=NONE
```

Comment:
guibd=#3f4b52 does not seem to match ctermbg=59, according to https://upload.wikimedia.org/wikipedia/en/1/15/Xterm_256color_chart.svg. Which one did you mean? I'm guessing 59, since 3f4b52 does not correspond to anything in the xterm palette. In this case, bgcolor should be 5f5f5f?